### PR TITLE
Update details.php

### PR DESCRIPTION
**Pull request by jirawat050**

### CODE REVIEW

File: www/details.php [hunk 1]
```diff
@@ -359,9 +359,7 @@ function handleRequestHash() {
                 stepNum = 1;
                 doExpandAll = true;
             }
-            if (stepNum <= 0) {
```
Comment (pmeenan) [inline www/details.php]: Why did you remove this?

Comment (pmeenan) [inline www/details.php]: Why did you remove this?

Comment (jirawat050) [reply www/details.php]: sorry i Hit wrong

**Comment by jirawat050** [comment]
i do change to finish
